Resources/config/services.xml

```diff
@@ -7,6 +7,9 @@
         <parameter key="bazinga_geocoder.geocoder.class">Geocoder\Geocoder</parameter>
         <parameter key="bazinga_geocoder.geocoder.loggable_class">Bazinga\Bundle\GeocoderBundle\Geocoder\LoggableGeocoder</parameter>
         <parameter key="bazinga_geocoder.geocoder.adapter.class">Geocoder\HttpAdapter\CurlHttpAdapter</parameter>
+        <parameter key="bazinga_geocoder.geocoder.adapter.geoip2.class">Geocoder\HttpAdapter\GeoIP2Adapter</parameter>
+        <parameter key="bazinga_geocoder.geocoder.adapter.geoip2_database.provider.class">GeoIp2\Database\Reader</parameter>
+        <parameter key="bazinga_geocoder.geocoder.adapter.geoip2_database.provider.filename"></parameter>
```

Comment:
I think you don't need to declare it here first, because you have already `on-invalid=null` down there.